ConfigurationSystem/Client/Utilities.py

```diff
@@ -589,4 +589,24 @@ def getElasticDBParameters( fullname ):
     dbPort = int( result['Value'] )
   parameters[ 'Port' ] = dbPort
 
-  return S_OK( parameters )
\ No newline at end of file
+  result = gConfig.getOption( cs_path + '/User' )
+  if not result['OK']:
+    # No individual port number found, try at the common place
+    result = gConfig.getOption( '/Systems/NoSQLDatabases/User' )
+    if result['OK']:
+      dbuserName = result['Value']
+  else:
+    dbuserName = result['Value']
+  parameters[ 'User' ] = dbuserName
+
+  result = gConfig.getOption( cs_path + '/Password' )
+  if not result['OK']:
+    # No individual port number found, try at the common place
+    result = gConfig.getOption( '/Systems/NoSQLDatabases/Password' )
```

Comment:
no, because we are using different password and user. Because EL is provided by IT better to not give them our password. They generated the password for us.